js/dropdowns.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
-/* global require, module, document */
-
 var $ = require('jquery');
-var perfectScrollbar = require('perfect-scrollbar/jquery')($);
+require('perfect-scrollbar/jquery')($);
```

Comment:
D'oh!  Not sure why I got that flipped around, but thank you for clarifying.  Basically I was just double checking to see that it truly didn't need to be referenced directly in the code, but it sounds it doesn't.